ci-operator/jobs/openshift/odo/openshift-odo-main-periodics.yaml

```diff
@@ -20,8 +20,8 @@ periodics:
       job_states_to_report:
       - failure
       - error
-      report_template: 'Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> @here '
+      report_template: <!subteam^SKUMSMU0N> Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
+        <{{.Status.URL}}|View logs>
```

Comment:
I don't understand how this will notify odo-team alias of the failure. But if you're sure it will, I don't mind doing lgtm on this PR.

Reply:
While searching for solution for `@here` I found of the team is using [this](https://github.com/openshift/release/blob/master/ci-operator/jobs/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master-periodics.yaml#L23) approach to notify the team alias on their `#forum-appsvc-notify` channel. 
In this PR I am trying to replicate that and use it for `odo-notification` channel.

Reply:
I think this should work. So we should move forward with it.

Reply:
So `<!subteam^SKUMSMU0N>` would convert to `@odo-team`? I don't understand how. Anyway, I'll lgtm since you're sure.